.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,38 @@
+name: CI/CD
+
+on: [push, workflow_dispatch]
+
+env:
+  DOCKER_BUILDKIT: 1
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.3.4
+
+      - name: Build
+        run: |
+          docker build --pull -t kooldev/php:8.0-nginx-swoole 8.0-nginx-swoole
```

Comment:
you didn't use the latest from oci8, right?

Can you update this CI to use matrix as @fabriciojs added here? https://github.com/kool-dev/docker-php-oci8/commit/78c1677eecdb82f68b7b7af339654455244dca57

Reply:
Right. i will do this update.